docs/modules/ROOT/nav.adoc

```diff
@@ -1,4 +1,11 @@
 include::user:ROOT:partial$shared-nav.adoc[]
 
+.AppCat
+* xref:appcat/availability.adoc[]
+
 .Object Storage Buckets
 * xref:object-storage/how-to.adoc[]
```

Comment:
Maybe rename to `create.adoc`?

Reply:
I don't see the reasoning behind this rename? The page is about what's available where and not about creating stuff?

Reply:
The pages' title is "Create an Object Storage Bucket" and it would make sense to not have everything on one page, when now having multiple pages for "jobs to be done".